src/frontend/src/instance.rs

```diff
@@ -447,6 +447,17 @@ impl FrontendInstance for Instance {
     }
 }
 
+impl Instance {
```

Comment:
I try to add `#[cfg(test)]` here. However, `tests-integration/tests` does not under the `#[cfg(test)]`.

Reply:
I think you can invoke `start` method in `FrontendInstance`.